test/systemtest/utils.py

```diff
@@ -0,0 +1,428 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import difflib
+import filecmp
+import inspect
```

Comment:
Some unused imports here (difflib, filecmp, inspect and select).

Later on, there are some comparisons (using `==`) with `None`. Maybe run flake8 on the file to catch them and clean them up?